lib/token_validator/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module TokenValidator
-  VERSION = '0.4.0'
+  VERSION = '0.4.1'
```

Comment:
Probably more than just a patch update. I would bump the minor version (i.e. `0.5.0`)

Reply:
👍